proxy/src/http/discovery.rs

```diff
@@ -0,0 +1,52 @@
+//! Endpoints for discovery feed.
+
+use warp::document::{self};
+use warp::{path, Filter, Rejection, Reply};
+
+/// `GET /discovery`
```

Comment:
If this is the endpoint to discover projects it should live under `projects` and doesn't require its own mount pont.

Reply:
I think that could make sense - I guess the argument for having a `discovery/` endpoint is that the discovery process might extend to other users or non-project entities in the future. @cloudhead what do you think?

Reply:
That will probably be the case, which would in turn mean that we have `/discover` endpoints under the respective mount points, e.g. `users` and so on. We should think along the lines of resources and how endpoints expose these resources. If one wants to discover a certain type of entity the least amount of surprise will be to find the endpoint to do so under the namespace of that entity.